src/scripts/Notifier.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint @typescript-eslint/camelcase: off */
```

Comment:
Why turn this off?

Reply:
I don't think we can control the names of those parameters for notifications,

```javascript
            allow_dismiss: true,
            newest_on_top: true,
            z_index: 1050,
            url_target: '_blank',
            mouse_over: null,
            icon_type: 'class',
```

It's still turned on for every other file though, just not this one.

Reply:
Ah fair point, but does it actually trigger there? You'd expect for it to only trip at the definitions

Reply:
You would think so, but unfortunately it does detect it.